handlers/system.go

```diff
@@ -0,0 +1,14 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func RegisterSystem() {
+	http.HandleFunc("/_status", systemStatus)
+}
+
+func systemStatus(w http.ResponseWriter, r *http.Request) {
```

Comment:
can we check if the sqs server is running?

Reply:
There is an endpoint checking server status
curl http://localhost/server